### PR TITLE
Execute click on dropdown select

### PR DESCRIPTION
**Pull request by Larspolo**
## Description
Execute click on dropdown select

### CODE REVIEW

File: src/vue/src/components/assets/DropdownButton.vue [hunk 1]
```diff
@@ -115,6 +115,7 @@ export default {
         emitChangeOption (option) {
             this.isOpen = false
             this.$emit('change-option', option)
+            this.$emit('click', option)
```
Comment (engelhamer) [inline src/vue/src/components/assets/DropdownButton.vue:118]: How does this work timing-wise? Could the change-option listener be not completed when this fires?

**Review by engelhamer — COMMENTED** [review]
How

File: src/vue/src/components/assets/DropdownButton.vue (hunk 1 above)
Comment (Larspolo) [reply src/vue/src/components/assets/DropdownButton.vue:118]: Think so

Comment (mjvkeulen) [reply src/vue/src/components/assets/DropdownButton.vue:118]: The events should be processed in order: options change should be handled before the click, there is no concurrency at work here.

But how to truly show this idk

Comment (Larspolo) [reply src/vue/src/components/assets/DropdownButton.vue:118]: Yeah, its ordered

**Comment by engelhamer** [comment]
Can be closed as we are opting for another solution?

**Pull request closed without merging**